README.md

```diff
@@ -277,3 +280,42 @@ topic naming convention:`<prefix>_entitlement`
   "applicationId": "application1-1.2.3"
 }
 ```
+
+## FAQ
+
+### How ignoreErrors affects tenant entitlement?
+
+_If `ignoreErrors` is set to `true` then the rollback operation is enabled if 1 or more stages failed.
```

Comment:
And if ignoreErrors is set to false - there is no rollback?

Reply:
Fixed typo and extended this section